{
  "comments": [
    {
      "key": {
        "uuid": "0107f113_331340d5",
        "filename": "validators/src/CaasValidation.py",
        "patchSetId": 2
      },
      "lineNbr": 193,
      "author": {
        "id": 183
      },
      "writtenOn": "2019-09-26T19:27:05Z",
      "side": 1,
      "message": "imagine you get this error as a user. you will have no idea what this refers to\nmention explicitly the exact name of the fault parameter\n\"CIDR\" is not an integer, it is a notation. you are looking for subnet size, or netmask",
      "revId": "19aa748f41d486dc72af65b054c5aa432d1803df",
      "serverId": "4cada01b-0314-4fe0-84a1-f9ddc89863f9",
      "unresolved": true
    }
  ]
}